### DISCUSSION TITLE
Place where comments are aggregated across deployments

### DISCUSSION THREAD

**Discussion by thomasmattheussen** [vercel/community#1074] [Idea]
Great job on the new comments feature!

One thing that seems missing to me is a nice overview where all comments are aggregated across deployments. I'm thinking of an extra tab in your project's dashboard for example.

**Comment by amyegan** [comment]
Thanks for reaching out @thomasmattheussen! Glad you're enjoying the comments feature. I'm sharing your suggestion with the team 😄

**Comment by gdborton** [comment]
@thomasmattheussen We're already working getting more feedback information on the dashboard, stay tuned!
